src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -1065,6 +1089,24 @@ internal sealed class Options : TransformInputBase
             [Argument(ArgumentType.AtMostOnce, HelpText = "Learning rate to use during optimization.", SortOrder = 12)]
             public float LearningRate = 0.01f;
 
+            /// <summary>
+            /// Whether early stopping technique should be used when accuracy stops improving.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Whether early stopping technique should be used when accuracy stops improving.", SortOrder = 15)]
+            public bool EnableEarlyStopping = true;
```

Comment:
Why is the default for this and `MinDelta` different than what it is in the public API? That is confusing.

Reply:
Similar behaviour exists for the Epoch option. Basically, the MinDelta can be 0.0, but a value of 0.01 works well in most cases, and hence the ImageClassificationEstimator is initialized with 0.01f but the option itself is set to 0. I can make both values same(0.01f) if it is indeed confusing to the users.

Reply:
Similar behaviour exists for the Epoch option. Basically, the MinDelta can be 0.0, but a value of 0.01 works well in most cases, and hence the ImageClassificationEstimator is initialized with 0.01f but the option itself is set to 0. I can make both values same(0.01f) if it is indeed confusing to the users.

---
In reply to: [327290199](https://github.com/dotnet/machinelearning/pull/4237#discussion_r327290199) [](ancestors = 327290199)